pkg/cmd/inspect.go

```diff
@@ -219,6 +222,19 @@ func (o *InspectOptions) Run() error {
 				continue
 			}
 		}
+
+		// save other related-object data for each clusteroperator
+		if len(otherObjs) == 0 {
+			log.Printf("unable to find any non-namespace related objects to clusteroperator/%s. Will not perform further data collection...\n", info.Name)
+		}
+		//for _, relatedObj := range otherObjs {
+		//	if len(relatedObj.Namespace) == 0 {
+		//		o.gatherClusterOperatorRelatedObjectData(path.Join(o.baseDir, "/cluster-scoped-resources", "/"+relatedObj.Group, "/"+relatedObj.Resource), relatedObj)
+		//		continue
+		//	}
+		//
+		//	o.gatherClusterOperatorRelatedObjectData(path.Join(o.baseDir, "/namespaces", "/"+relatedObj.Namespace, "/"+relatedObj.Resource), relatedObj)
```

Comment:
we'll need to store Version for every configv1.ObjectReference in order to fetch and store the resource using a dynamic client (if this patch makes sense)